packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -340,11 +343,71 @@ def test_vip(dcos_api_session,
                 pm_app.purge(dcos_api_session)
     assert not errors
 
+    containers = list(marathon.Container) + [Container.POD]
+    return [(container, vip_net, proxy_net)
+            for container in containers
+            for vip_net in list(marathon.Network)
+            for proxy_net in list(marathon.Network)]
+
+
+@pytest.mark.parametrize('container', [marathon.Container.MESOS, Container.POD])
+@pytest.mark.parametrize('vip_net', [marathon.Network.USER])
+@pytest.mark.parametrize('proxy_net', [marathon.Network.USER])
+def test_calico_vip(dcos_api_session,
+                    container: marathon.Container,
+                    vip_net: marathon.Network,
+                    proxy_net: marathon.Network,
+                    ipv6: bool=False,
+                    with_port_mapping_app=True):
+    '''Test VIPs between the following source and destination configurations:
+        * containers: DOCKER, UCR and NONE
```

Comment:
Is this comment relevant to the current configuration of this test?

Reply:
copied from `test_vip`

Reply:
To achieve minimal change.
`test_vip` is not allowed to extend calico tests, so I just copied `test_vip`.
Actually I tried to refactor the `test_networking.py` a bit to be more extensible, but we have too many bug fixes in this file.

Reply:
> copied from test_vip

What I meant is that the description reflects the parameters of the `test_vip` but the parameters of `test_calico_vip` are different.

Reply:
I'd say the configurations are used to explain the combinations of scenarios we are going to test, not to explain the function parameter.
I will remove these not covered by this test.